5.2: Ledger repair functionality (#2110)
corda/corda-docs-portal: content/en/platform/corda/5.2/deploying-operating/deployment/deploying/_index.md

  sidecar.istio.io/inject: !!str true # explicitly enable Istio integration for all Corda pods
```

### Ledger Repair

> By default, the [Corda ledger repair functionality]({{< relref "../../../developing-applications/ledger/ledger-repair.md" >}}) runs every ten minutes. You can modify this schedule by setting the `net.corda.ledger.utxo.repair.schedulePeriod` system property for each database worker. This property is set in seconds. For example, to modify the schedule to every 15 minutes:

Reviewer: Related to the comment below...
```suggestion
By default, the [Corda ledger repair functionality]({{< relref "../../../developing-applications/ledger/ledger-repair.md" >}}) runs every ten minutes. You can modify this schedule by adding the `net.corda.ledger.utxo.repair.schedulePeriod` system property to the Java options for each database worker. This property is set in seconds. For example, to modify the schedule to every 15 minutes:
```

```yaml
workers:
  db:

>     javaOptions: "-Dnet.corda.ledger.utxo.repair.schedulePeriod=900"

Reviewer: As we have `-XX:MaxRAMPercentage=75` as the default value for this field, we should probably include it in the example; otherwise, they'll be unsetting it.
```suggestion
    javaOptions: "-XX:MaxRAMPercentage=75 -Dnet.corda.ledger.utxo.repair.schedulePeriod=900"
```

```

> The window of transactions included in the repair process and the length of time the process can run for is defined by fields in the[corda.ledger.utxo configuration section]({{< relref "../../config/fields/ledger-utxo.md" >}}).

Reviewer 2: ```suggestion
The window of transactions included in the repair process and the length of time the process can run for is defined by fields in the [corda.ledger.utxo configuration section]({{< relref "../../config/fields/ledger-utxo.md" >}}).
```
